styleguide/source/assets/js/tabs.js

```diff
@@ -40,27 +23,31 @@
         // Use currentTarget because target is sometimes the icon div.
         window.location.hash = e.currentTarget.hash;
         window.scroll({top: windowScrollY});
+        return false;
       });
 
-      // Scroll to when there is no has change on resource page tab links
-      $('.ama____page--resources__page-content a[href^="#"]').on('click', function (e) {
-        if (lastHash == window.location.hash) {
-          var hash = window.location.hash;
-          smoothScroll($('.ama__resource-tabs__nav'), $('.ama__resource-tabs__nav a[href="' + hash + '"]'));
-        }
-      });
 
       //Simulate click event on actual simpleTabs tab from mobile drop down.
       $('.ama__tabs-navigation--mobile select').on("selectmenuchange", function (event, ui) {
         var selectedValue = ui.item.value;
         $('a[href="#' + selectedValue + '"]').click();
       });
 
+      // When clicking an inline resource page link referencing a tab, open referenced tab.
+      $('.ama__resource-link--inline, .ama__page--resource__resource-link').on('click', function (e) {
+        e.preventDefault();
+        var $tabs = $('.ama__resource-tabs');
+        var $clickedObj = $(this);
+        var linkHash = this.getAttribute("href");
```

Comment:
jQuery has the `$().attr('href')` way to do this. I think either is fine, but since you're already using jQuery, that might be easier to follow.

Reply:
Since I didn't need the jQuery object elsewhere in my logic I opted to just use vanilla js and the already present `this` var.